pyseir/deployment/demographic_mapper.py

```diff
@@ -0,0 +1,719 @@
+            self.parameters['hospitalization_length_of_stay_general']
+        hospital_recovery_inflow_rates['HICU'] = (1 - mortality_inflow_rates['HICU']) * (1 - self.parameters[
+            'fraction_icu_requiring_ventilator']) \
+            / self.parameters['hospitalization_length_of_stay_icu']
+        hospital_recovery_inflow_rates['HVent'] = \
+            (1 - np.maximum(mortality_inflow_rates['HVent'],
+                            self.parameters['mortality_rate_from_ICUVent'])) \
+            / self.parameters['hospitalization_length_of_stay_icu_and_ventilator']
+
+        return hospital_recovery_inflow_rates
+
+    def _age_specific_IHR_by_hospitalization(self, measure_unit):
+        """
+        Calculate age specific hospitalization rates among infected
+        population (IHR) for a given measure unit by types of
+        hospitalization.
+
+        When unit is per capita, calculates the probability that an infected
+        person (asymptomatic or symptomatic) ultimately gets admitted to
+        hospital:
+            probability of being symptomatic * rate of hospitalized / (rate of
+            hospitalized + rate of recovery without hospitalization)
+
+        When unit is per capita day, calculate the probability that an
+        infected person gets admitted to hospital per day:
+           new hospitalization / (asymptomatic + symptomatic infections)
+
+        Parameters
+        ----------
+        measure_unit: CovidMeasureUnit
+            Unit of covid measure
+
+        Returns
+        -------
+        IHR: dict
+            Contains:
+            - <hospitalization>: dict
+              - <measure_unit>:  np.array
+                Rates of admission to hospital of infected population
+                by types of hospitalization.
+              hospitalization include: HGen, HICU, HVent.
+        """
+
+        hospital_inflow_rates = self._reconstruct_hospitalization_inflow_rates()
+
+        fraction_of_symptomatic = self.predictions_by_age['I'] / (self.predictions_by_age['I']
+                                                                + self.predictions_by_age['A'])
+        IHR = defaultdict(dict)
+        if measure_unit is CovidMeasureUnit.PER_CAPITA_DAY:
+            HR = hospital_inflow_rates
+            for key in HR:
+                IHR[key][measure_unit.value] = HR[key][:, np.newaxis] * fraction_of_symptomatic
+
+        elif measure_unit is CovidMeasureUnit.PER_CAPITA:
+            total_rate_out_of_I = self.parameters['delta']
+            for key in hospital_inflow_rates:
+                total_rate_out_of_I += hospital_inflow_rates[key]
+
+            HR = {}
+            for key in hospital_inflow_rates:
+                HR[key] = np.tile(hospital_inflow_rates[key] / total_rate_out_of_I,
+                                  (len(self.parameters['t_list']), 1)).T
+            for key in HR:
+                IHR[key][measure_unit.value] = HR[key] * fraction_of_symptomatic
```

Comment:
Add an `else` statement that raises a `ValueError` (stating that `measure_unit` isn't implemented)